src/zrc/api/tests/test_zaken.py

```diff
@@ -543,6 +548,50 @@ def test_sort_startdatum(self):
         self.assertEqual(data[1]["startdatum"], "2019-02-01")
         self.assertEqual(data[2]["startdatum"], "2019-01-01")
 
+    def test_zaak_eigenschappen_as_inline(self):
+        zaak1 = ZaakFactory.create(zaaktype=ZAAKTYPE)
+        zaak2 = ZaakFactory.create(zaaktype=ZAAKTYPE)
+        eigenschap1, eigenschap2 = ZaakEigenschapFactory.create_batch(2, zaak=zaak1)
+
+        url = reverse(Zaak)
+
+        response = self.client.get(url, **ZAAK_READ_KWARGS)
+
+        results = response.data["results"]
+        self.assertEqual(results[0]["eigenschappen"], [])
+
+        eigenschappen = [
```

Comment:
This variable is not used an overwritten in line 591, right? Probably better to remove it alltogether

Reply:
I don't even know what I thought